forgit.plugin.zsh

```diff
@@ -52,6 +52,12 @@ forgit::diff() {
 forgit::add() {
     forgit::inside_work_tree || return 1
     local changed unmerged untracked files opts
+    # Add files if passed as arguments
+    [[ $# -ne 0 ]] && {
+        eval "git add $@"
```

Comment:
Hi @carlosedp I don't see why `eval` is needed here, can you explain it? Or we just use `git add "$@" instead?

Reply:
Sure, I just wanted to keep consistent with other functions. Will push the change.

Reply:
LGTM. Thanks for your contribution!